web/packages/plugins/apps/analysis-platform/src/modules/Analysis/components/AnalysisCard.tsx

```diff
@@ -148,21 +148,23 @@ const AnalysisCard = (props: AnalysisCardProps) => {
           </div>
         </div>
         <Card.Actions>
-          {'task' in analysis && Object.keys(analysis.task).length > 0 && (
-            <>
-              <RunAnalysisButton
-                analysis={analysis}
-                addJob={addJob}
-                jobs={jobs}
-              />
-              <Tooltip title={'Not implemented'}>
-                <Button style={{ width: 'max-content' }} disabled>
-                  Configure schedule
-                  <Icons name="time" title="time" />
-                </Button>
-              </Tooltip>
-            </>
-          )}
+          {hasExpertRole(tokenData) &&
```

Comment:
looks like hasExpertRole(tokenData) is present on all components inside <Card.Actions>. Is it possible to move the hasExpertRole to conditionally render the entire <Card.Actions> block?

Reply:
Indeed! Fixed!